backend/battles/views.py

```diff
@@ -0,0 +1,33 @@
+from django.shortcuts import get_object_or_404
+from django.views.generic import CreateView, DetailView, ListView
+
+from battles.forms import BattleForm
+from battles.models import Battle
+from users.models import User
+
+
+class BattleCreate(CreateView):
+    template_name = 'battles/battle_form.html'
+    form_class = BattleForm
+
+    def get_form_kwargs(self):
+        kwargs = super(BattleCreate, self).get_form_kwargs()
+        kwargs['user'] = User.objects.first()
+        return kwargs
+
+    def form_valid(self, form):
+        creator = get_object_or_404(User, email='jaenia@vinta.com.br')
```

Comment:
How are tests passing if there's no user with this email? You are probably missing something. I was expecting you would use the same logic as in `get_form_kwargs`

Reply:
Yes, I agree.